serialization.md

```diff
@@ -1,7 +1,7 @@
 # OCI Image Serialization
 
 An *Image* is an ordered collection of root filesystem changes and the corresponding execution parameters for use within a container runtime.
-This specification outlines the format of these filesystem changes and corresponding parameters and describes how to create and use them for use with a container runtime and execution tool.
+This specification outlines the JSON format describing images for use with a container runtime and execution tool and its relationship to filesystem change sets, described in [Layers](layer.md).
 
 ## Terminology
```

Comment:
@wking Layer has very specific to this version of the specification and config. For the most part, the meaning of a layer at the higher-level is largely agnostic. At this level, their meaning is narrow and part of their relationship with the image configuration.

Read through it again.

Pulling this detail up doesn't help for contextual understanding or clarity.

Either way, this is massively outside the scope of this PR. You may submit another PR that removes, but I won't LGTM, as it creates confusion, ambiguity and indirection that makes meaning impossible to interpret.